plugins/modules/elastic_expedient_settings.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/python
+# Copyright 2021 Expedient
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+DOCUMENTATION='''
+
+module: elastic_settings_update
+
+author: Ian Scott
+
+short_description: Update Elasticsearch settings.
+
+description: 
+  - Update Elasticsearch settings such as routing, metadata, etc.
+
+requirements:
+  - python3
+
+options:
+      host: ECE Host or Deployment Host
+      port: ECE Port or Deployment Port
+      username: ECE Username or Deployment Username
+      password: ECE Password or Deployment Password
+      deployment_info: (when using ECE host:port and credentials)
+        deployment_id: ECE Deployment ID
+        deployment_name: ECE Deployment Name
+        resource_type: kibana
+        ref_id: REF ID for kibana cluster, most likely main-kibana
+        version: Deployment Kibana Version
```

Comment:
can you update this to the normal ansible `DOCUMENTATION` format?

Reply:
see the notes on the other `DOCUMENTATION` string and the docs here - https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html but we want at least name, description, short_description, and then type, description, and whether or not the option is required for each option.